go-controller/pkg/util/ovs.go

```diff
@@ -229,9 +233,70 @@ func RunOVNNbctlWithTimeout(timeout int, args ...string) (string, string,
 		}
 	}
 
+	return cmdArgs
+}
+
+var daemonRunning bool
```

Comment:
You might need to lock daemonRunning since we'll likely be accessing it from multiple goroutines. Also we'll need ot update testcases, run 'make check' to see what needs to happen there.